requirements.txt

```diff
@@ -3,7 +3,7 @@ django_readonly_field
 jsonschema>=3.0
 libcomps~=0.1.15
 productmd~=1.33.0
-pulpcore>=3.13.0.dev
+pulpcore>=3.14.2
```

Comment:
Does this need to be the actual final values or is it still changed during the process of creating the release?

Reply:
This needs to be the actual final values.